File: JavaScript/1005.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

a = parseFloat(lines.shift()) * 3.5
b = parseFloat(lines.shift()) * 7.5
soma_pesos = 3.5 + 7.5

media = (a + b) / soma_pesos

console.log(`MEDIA = ${media.toFixed(5)}`)

File: C/1005.c
#include <stdio.h>

int main() {

  double a, b, media;
  float p1 = 3.5;
  float p2 = 7.5;

  scanf("%lf", &a);
  scanf("%lf", &b);
  media = ((a * p1) + (b * p2)) / (p1 + p2);

  printf("MEDIA = %.5lf\n", media);
  
  return 0;
  
}
